exec/java-exec/src/test/java/org/apache/drill/exec/vector/complex/writer/TestJsonReader.java

```diff
@@ -804,4 +804,59 @@ public void testConvertFromJson() throws Exception {
         .baselineValues("2", "abc")
         .go();
   }
+  @Test // DRILL-7821
```

Comment:
@mkusnir Minor thing... Missing the new line between the  `}` and `@Test`.
You can push a revision again (please squash commits), then ready to merge, thanks

Reply:
@luocooong Sorry about that, just pushed the fix. Also, thanks for the rebase instructions you gave me on #2170.